src/pages/Foundations/AspectRatios/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: 'Aspect ratios'
+order: 10
+---
+
+> The aspect ratio of an image is **the proportional relationship between its width and its height**. Aspect ratios are commonly written as a formula of width to height, like “16:9” or “4:3". The first value corresponds to the width, and the second to the height. <br>
+> For example, an image with an aspect ratio of 1:1 is a square. So, the width and the height are identical, like 80px x 80px.
+
+## Utility
+
+For designers, by using aspect ratio for images, and applying it on yours products, you will be able to create consistency between all the pictures.<br>
```

Comment:
I don't thinks this all section is useful. I would just remove it, or add something very small like : 

"aspect ration bring consistent image sizes across product and viewports"